ConsoleApp/BankAccount.cs

```diff
@@ -0,0 +1,64 @@
+namespace ConsoleApp
+{
+    public class BankAccount
+    {
+        private int Balance {get; set; }
+        private string[] OperationsHistory;
+        public int Id { get; init; }
+        public BankAccount(int id)
+        {
+            Id = id;
+            Balance = 0;
+            OperationsHistory = Array.Empty<string>();
+        }
+
+        public void ShowBalance()
+        {
+            Console.WriteLine($"Your balance: {Balance}");
+        }
+        public void ShowOperationsHistory()
```

Comment:
instead of write to console, it's better to return copy of operations so use of code can decide what to do next